tools/debian/cockpit-ws.install

```diff
@@ -14,6 +14,7 @@ ${env:deb_systemdsystemunitdir}/system-cockpithttps.slice
 ${env:deb_pamlibdir}/security/pam_ssh_add.so
 ${env:deb_pamlibdir}/security/pam_cockpit_cert.so
 usr/lib/tmpfiles.d/cockpit-tmpfiles.conf
+usr/lib/sysusers.d/cockpit-sysusers.conf
```

Comment:
So this just works in Debian because cockpit-session is chmoded in the postinst script, instead of the rpm. Is that legit for RPM as well? This would be much more palatable to me.

Reply:
... I'd sure love to move cockpit-session over to socket activation and drop this bit...